examples/counter/tests/single_chain.rs

```diff
@@ -13,12 +13,13 @@ use linera_sdk::test::TestValidator;
 /// which is then checked.
 #[tokio::test(flavor = "multi_thread")]
 async fn single_chain_test() {
-    let (validator, bytecode_id) = TestValidator::with_current_bytecode().await;
+    let (validator, bytecode_id) =
+        TestValidator::with_current_bytecode::<counter::CounterAbi, (), u64>().await;
```

Comment:
We could consider (maybe as a separate PR) adding another trait, maybe `App` or `BytecodeAbi` or something:

```rust
trait App {
    type Abi;
    type InitializationArgument;
    type Parameters;
}
```

Then `with_current_bytecode` and `BytecodeId` would only have one generic argument.

(But maybe it's not worth adding another trait; just a thought.)